scripts/setup_respeaker.sh

```
@@ -55,7 +64,11 @@ apt-get -y install dkms git i2c-tools libasound2-plugins
 
 kernels=($(ls /boot/vmlinuz-* | sort -V | sed 's|/boot/vmlinuz-||'))
 
-install_respeaker "./" "seeed-voicecard"
+if [ install_respeaker "./" "seeed-voicecard" -eq 0]; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)